src/commands/account/delete_key/mod.rs

```diff
@@ -33,21 +33,25 @@ impl DeleteKeyCommandContext {
 
 impl From<DeleteKeyCommandContext> for crate::commands::ActionContext {
     fn from(item: DeleteKeyCommandContext) -> Self {
+        let on_after_getting_network_callback: crate::commands::OnAfterGettingNetworkCallback =
+            std::sync::Arc::new(move |_network_config| {
+                Ok(crate::commands::PrepopulatedTransaction {
+                    signer_id: item.owner_account_id.clone().into(),
+                    receiver_id: item.owner_account_id.clone().into(),
```

Comment:
Do we need `.into()` here? In other places, you don't use it.

Reply:
Done